### DISCUSSION TITLE
Removing and Adding new entity labels to en_core_web_lg

### DISCUSSION THREAD

**Discussion by aniyyanz08** [explosion/spaCy#9494] [Help: Model Advice] [answered]
I found that en_core_web_lg has named recognition entity labels as CARDINAL, DATE, EVENT, FAC, GPE, LANGUAGE, LAW, LOC, MONEY, NORP, ORDINAL, ORG, PERCENT, PERSON, PRODUCT, QUANTITY, TIME, WORK_OF_ART. 

Out of these labels i need DATE, EVENT, ORG, PERSON, TIME, WORK_OF_ART. Other entity labels from the above are not needed for my application. Also i need few other entity labels to be included to this. Is it possible to remove few of the existing labels and add few other new labels to this list.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi, @aniyyanz08

> Out of these labels i need DATE, EVENT, ORG, PERSON, TIME, WORK_OF_ART. Other entity labels from the above are not needed for my application. 

You can safely ignore or filter the labels you don't need: 

```python
doc = nlp(SOME_TEXT)
ents = [ent for ent in list(doc.ents) if ent.label_ not in LIST_OF_FILTERS]
```

>  Also i need few other entity labels to be included to this.

You can check the [multiple NER demo](https://github.com/explosion/projects/tree/v3/tutorials/ner_double) on how to do that. From here you can combine multiple components to include your new labels
